packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -64,6 +64,7 @@ abstract class CameraPlatform extends PlatformInterface {
   /// [imageFormatGroup] is used to specify the image formatting used.
   /// On Android this defaults to ImageFormat.YUV_420_888 and applies only to the imageStream.
   /// On iOS this defaults to kCVPixelFormatType_32BGRA.
+  /// On Web this parameter is currently not supported.
```

Comment:
@ditman Should this be reverted for now?

Reply:
Correct, let's add a pass documenting all the web stuff later on, since we need to publish a new version of the platform_interface so the new docs show up in pub.